physionet-django/sso/views.py

```diff
@@ -68,7 +68,7 @@ def sso_register(request):
     """
     user = request.user
     if user.is_authenticated:
-        return redirect('project_home')
+        return redirect('home')
```

Comment:
```suggestion
        return redirect('project_home')
```

I'd say it's better to redirect to `project_home` which is "user specific" so it's immediately clear to the user that they were logged in, since they can see their projects.